Pharmacy Database Project-csharp version/Domain/Services/Implementations/PaymentService.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Domain.Models;
+using Domain.Repositories.Interface;
+using Domain.Services.Interfaces;
+
+namespace Domain.Services.Implementations;
+
+public class PaymentService : IPaymentService
+{
+    private readonly IRepository<PaymentDomain, int> _paymentRepository;
```

Comment:
same for all services

```suggestion
    private readonly IPaymentRepository _paymentRepository;
```